### ISSUE TITLE
[1.0] Assertion failed with build EAP 14.1 (140.2110.5)

### ISSUE THREAD

**Issue by dlsniper** [go-lang-plugin-org/go-lang-idea-plugin#1244]
I've installed the latest EAP for IDEA and I've got this one.

```java
2015-02-03 21:01:35,090 [  19382]  ERROR - rations.RunConfigurationModule - IntelliJ IDEA (Cassiopeia) IU-140.2110.5  Build #IU-140.2110.5 
2015-02-03 21:01:35,090 [  19382]  ERROR - rations.RunConfigurationModule - JDK: 1.8.0_31 
2015-02-03 21:01:35,090 [  19382]  ERROR - rations.RunConfigurationModule - VM: Java HotSpot(TM) 64-Bit Server VM 
2015-02-03 21:01:35,090 [  19382]  ERROR - rations.RunConfigurationModule - Vendor: Oracle Corporation 
2015-02-03 21:01:35,090 [  19382]  ERROR - rations.RunConfigurationModule - OS: Linux 
2015-02-03 21:01:35,090 [  19382]  ERROR - rations.RunConfigurationModule - Last Action:  
2015-02-03 21:01:35,091 [  19383]  ERROR - rations.RunConfigurationModule - Assertion failed 
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:144)
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:153)
	at com.intellij.execution.configurations.RunConfigurationModule.readExternal(RunConfigurationModule.java:61)
	at com.intellij.execution.configurations.ModuleBasedConfiguration.readModule(ModuleBasedConfiguration.java:72)
	at com.goide.runconfig.GoRunConfigurationBase.readExternal(GoRunConfigurationBase.java:59)
	at com.intellij.execution.impl.RunnerAndConfigurationSettingsImpl.readExternal(RunnerAndConfigurationSettingsImpl.java:371)
	at com.intellij.execution.impl.RunManagerImpl.loadConfiguration(RunManagerImpl.java:842)
	at com.intellij.execution.impl.RunManagerImpl.loadState(RunManagerImpl.java:698)
	at com.intellij.execution.impl.RunManagerImpl.loadState(RunManagerImpl.java:50)
	at com.intellij.openapi.components.impl.stores.ComponentStoreImpl.a(ComponentStoreImpl.java:245)
	at com.intellij.openapi.components.impl.stores.ComponentStoreImpl.initComponent(ComponentStoreImpl.java:73)
	at com.intellij.openapi.project.impl.ProjectImpl.initializeComponent(ProjectImpl.java:204)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter$1.getComponentInstance(ComponentManagerImpl.java:567)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentConfigComponentAdapter.getComponentInstance(ComponentManagerImpl.java:611)
	at com.intellij.util.pico.DefaultPicoContainer.getLocalInstance(DefaultPicoContainer.java:245)
	at com.intellij.util.pico.DefaultPicoContainer.getComponentInstance(DefaultPicoContainer.java:211)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.createComponent(ComponentManagerImpl.java:126)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:117)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:88)
	at com.intellij.openapi.project.impl.ProjectImpl.init(ProjectImpl.java:295)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:323)
	at com.intellij.openapi.project.impl.ProjectManagerImpl.access$700(ProjectManagerImpl.java:86)
	at com.intellij.openapi.project.impl.ProjectManagerImpl$12.compute(ProjectManagerImpl.java:633)
	at com.intellij.openapi.project.impl.ProjectManagerImpl$12.compute(ProjectManagerImpl.java:629)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$5.run(ProgressManagerImpl.java:380)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$TaskRunnable.run(ProgressManagerImpl.java:618)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$7.run(ProgressManagerImpl.java:421)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl$3.run(ProgressManagerImpl.java:193)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.a(ProgressManagerImpl.java:282)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:232)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.runProcess(ProgressManagerImpl.java:180)
	at com.intellij.openapi.application.impl.ApplicationImpl$10$1.run(ApplicationImpl.java:626)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:408)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
```

**Comment by dmitry-vovk** [comment]
I confirm. Neither 1.0.0.alpha#11 nor 0.9.16-alpha.9 are usable with the IU-140.2110.5.

**Comment by dlsniper** [comment]
I've noticed something strange. If I run the plugin from IDEA, it doesn't complain the issue, but if I install the plugin and restart IDEA, then the error happens.

**Comment by zolotov** [comment]
> If I run the plugin from IDEA, it doesn't complain the issue, but if I install the plugin and restart IDEA, then the error happens.

Maybe you run plugin it 139.* SDK.

Anyway, I've recreated the exception and know what leads to it. It turned out that we just can't handle this issue for both versions 14 and 14.1. We should maintain two different plugin versions for that and I'm not ready to it yet :)

**Comment by dlsniper** [comment]
Oh, maintaining two versions would definitely be a problem but wouldn't it be easier to revert the changes that introduce the bug to version 15 of IDEA? I don't expect to be the only plugin to have this problem, no?

**Comment by zolotov** [comment]
We're discussing the opportunity to fix this on IntelliJ Platform side.

**Comment by zolotov** [comment]
Will be fixed in the next IDEA 14.1 EAP
